build/assets/scripts/pre-boot-tuning.sh

```diff
@@ -2,114 +2,26 @@
 
 set -euo pipefail
 
-non_iso_cpumask=""
-cpu_affinity=""
+SYSTEM_CONFIG_FILE="/etc/systemd/system.conf"
+SYSTEM_CONFIG_CUSTOM_FILE="/etc/systemd/system.conf.d/setAffinity.conf"
 
-get_reserved_cores() {
-    cores=()
-    while read part; do
-        if [[ $part =~ - ]]; then
-            cores+=($(seq ${part/-/ }))
-        elif [[ $part =~ , ]]; then
-            continue 
-        else
-            cores+=($part)
-        fi
-    done < <( echo ${NON_ISOLATED_CPUS} | tr ',' '\n' )
-}
-
-# $1 - 0 for irq balance banned cpus masking , 1 for non isolated cpus masking
-get_cpu_mask() {
-    if [ "$1" = "1" ]; then
-        mask=( 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 )
-    else
-        mask=( 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 ) 
-    fi    
-    get_reserved_cores
-    for core in ${cores[*]}; do
-        mask[$core]=$1
-    done
-    cpumaskBinary=`echo ${mask[@]}| rev`
-    cpumaskBinary=${cpumaskBinary//[[:space:]]/}
-    non_iso_cpumask=`printf '%08x\n' "$((2#$cpumaskBinary))"`
-}
-
-get_cpu_affinity() {
-    cpu_affinity=""
-    get_reserved_cores
-    for core in ${cores[*]}; do
-        cpu_affinity+=" $core"
-    done
-    echo "CPU Affinity set to $cpu_affinity"
-}
-
-# TODO - find a more robust approach than keeping the last timestamp
-RHCOS_OSTREE_PATH=$(ls -td /boot/ostree/*/ | head -1)
-RHCOS_OSTREE_BOOTLOADER_PATH=${RHCOS_OSTREE_PATH#"/boot"}
-INITRD_GENERATION_DIR="/root/initrd"
-INITRD_NEW_IMAGE="${RHCOS_OSTREE_PATH}/iso_initrd.img"
-
-OSTREE_VERSION=$(rpm-ostree status -b | grep Version | awk '{print $2}')
-CURRENT_ENTRY_FILE=$(grep -Rls ${OSTREE_VERSION} /boot/loader/entries/)
-
-if [ -f ${INITRD_NEW_IMAGE} ] && grep -qs "iso_initrd.img" ${CURRENT_ENTRY_FILE}; then
+if [ -f /etc/sysconfig/irqbalance ] && [ -f ${SYSTEM_CONFIG_CUSTOM_FILE} ] && grep -ls "IRQBALANCE_BANNED_CPUS=${NON_ISOLATED_CPU_MASK_INVERT}" /etc/sysconfig/irqbalance; then
```

Comment:
Can we move away from using the term "NON_ISOLATED" and use RESERVED and ISOLATED instead.